providers/implementations/kdfs/kbkdf.c

```diff
@@ -328,6 +353,16 @@ static int kbkdf_set_ctx_params(void *vctx, const OSSL_PARAM params[])
     if (p != NULL && !OSSL_PARAM_get_int(p, &ctx->use_l))
         return 0;
 
+    p = OSSL_PARAM_locate_const(params, OSSL_KDF_PARAM_KBKDF_R);
+    if (p != NULL) {
+        int new_r = 0;
+        if (!OSSL_PARAM_get_int(p, &new_r))
+            return 0;
+        if ((new_r != 8) && (new_r != 16) && (new_r != 24) && (new_r != 32))
```

Comment:
We generally leave out the parenthesis in conditionals like this.  Not a biggie either way.